README.md

```diff
@@ -37,6 +37,12 @@ Money.default_bank = MoneyOXR::Bank.new(
   cache_path: 'config/oxr.json',
   max_age: 86400
 )
+
+MoneyOXR::RatesStore.new({
+  app_id: 'abcd1234',
+  cache_path: 'config/oxr.json',
+  max_age: 86400
+}).load
```

Comment:
I'm not sure why this is here.  The MoneyOXR::Bank creates its own rates store.

Reply:
@edlebert thank you for bringing this to my attention, and it looks like you are right. The load method is not necessary. In [my explorations](https://github.com/robskrob/money-oxr-example) with your gem I notice that `money-oxr` makes an API call whenever `Money#exchange_to` or `Money#get_rate` is called -- if the json cache is not available or if the cache is stale. I have not tested the stale part my self.